src/main/java/amidst/gui/profileselect/LocalProfileComponent.java

```diff
@@ -1,7 +1,5 @@
 package amidst.gui.profileselect;
 
-import java.io.FileNotFoundException;
```

Comment:
Yes. IDEA and Eclipse both have the *organize imports* feature, however they seem to choose a different ordering. This is why I prefer to have all developers use a single set of tools. This is just an explanation, I saw you already fixed it for this pull request.